app/src/main/java/org/p2p/wallet/history/HistoryModule.kt

```diff
@@ -54,9 +54,13 @@ object HistoryModule : InjectionModule {
     private fun Module.dataLayer() {
         factory { HistoryRemoteRepository(compareApi = get()) } bind HistoryRepository::class
 
-        factory { ConfirmedTransactionParser }
-        factory { TransactionDetailsMapper(confirmedTransactionParser = get(), gson = get()) }
-        factory { HistoryTransactionMapper(userLocalRepository = get()) }
+        factory { TransactionDetailsEntityMapper() }
+        factory {
+            HistoryTransactionMapper(
+                userLocalRepository = get(),
+                historyTransactionConverter = HistoryTransactionConverter()
```

Comment:
Converter can be a object

Reply:
I don't think that's a good idea
Im injecting it because it helps to mock converter in unit tests